core/sequencer/sequencer_api.proto

```diff
@@ -75,8 +68,23 @@ message DefineRevisionsRequest {
 
 // DefineRevisionResponse contains information about freshly defined revisions.
 message DefineRevisionsResponse {
-  // outsanding_revisions a list of all the defined revisions which are not yet applied.
-  repeated int64 outstanding_revisions = 1;
+  // revisions is the list of newly defined revisions.
+  repeated int64 revisions = 2;
+  // Deprecated tags, do not reuse.
+  reserved 1;
+}
+
+// GetUnappliedRevisionsRequest contains information needed to query the set of
+// outstanding revisions.
+message GetUnappliedRevisionsRequest {
```

Comment:
`ListUnappliedRevisions`

Reply:
PageSize

Reply:
Not relevant anymore, as we decided to remove lists of revisions.